apps/app/src/views/routes/Settings/components/invites/InviteItem.tsx

```diff
@@ -0,0 +1,29 @@
+import { Badge, Td, Text, Tr } from '@chakra-ui/react';
+import { FragmentType, graphql, useFragment } from 'apps/app/src/gql';
+
+export const inviteFragment = graphql(/* GraphQL */ `
```

Comment:
I like that you're doing it this way to get the invite prop type 🙏, good example for the rest of the codebase